esx_service/utils/auth_data.py

```diff
@@ -561,10 +568,46 @@ def handle_upgrade_1_1_to_1_2(self):
                      """
             sql_script = script.format(DB_MAJOR_VER, DB_MINOR_VER)
             self.conn.executescript(sql_script)
+
+            logging.debug("handle_upgrade_1_1_to_1_2: update vms table Done")
+
+            # update the tenants table to set "default_datastore" to "__VM_DS" if "default_datastore" is ""
+            self.conn.execute("UPDATE OR IGNORE tenants SET default_datastore_url = ?  where default_datastore_url = ?", (auth_data_const.VM_DS_URL, "" ))
```

Comment:
nit: no need to pass "" when you can put it in the query directly 
self.conn.execute("UPDATE OR IGNORE tenants SET default_datastore_url = ?  where default_datastore_url = \\"\\"", (auth_data_const.VM_DS_URL) )